src/Microsoft.FeatureManagement.AspNetCore/AspNetCoreFeatureManagementBuilderExtensions.cs

```diff
@@ -44,5 +47,28 @@ public static IFeatureManagementBuilder UseDisabledFeaturesHandler(this IFeature
 
             return builder;
         }
+
+        /// <summary>
+        /// Adds the <see cref="DefaultHttpTargetingContextAccessor"/> to be used for targeting and registers the targeting filter to the feature management system.
```

Comment:
That's an internal type. It shouldn't show up in a public method summary. The compiler allows it?

Reply:
Since `DefaultHttpTargetingContextAccessor` is internal, users won't see it's method summary so some of those details would be useful here.

Reply:
Good point- how about:   
```suggestion
        /// Adds the default ASP.NET <see cref="ITargetingContextAccessor"/> to be used for targeting and registers the targeting filter to the feature management system.
```
?

Reply:
I would say something like.

`Enables the use of targeting within the application and adds a targeting context accessor that extracts targeting details from a request's HTTP context.`